src/screens/MVUMInspectorScreen.js

```diff
@@ -0,0 +1,37 @@
+import React, {Component} from 'react';
+import {StyleSheet, WebView, View} from 'react-native';
+import {connect} from 'react-redux';
+import Gallery from 'react-native-image-gallery';
+
+import _ from 'lodash';
+
+class MVUMInspectorScreen extends Component {
+
+    render() {
+        const {fillScreen} = styles;
+        const {selectedSite: {mvum}} = this.props;
+
+        return (
+            <View style={fillScreen}>
+                <WebView
+                    bounces={false}
+                    scrollEnabled={false}
+                    source={{ uri: `./${mvum}.pdf` }} />
+            </View>
+        );
+    }
+}
+
+const styles = StyleSheet.create({
+    fillScreen: {
+        flex: 1
+    }
+});
+
+function mapStateToProps(state) {
+    const {selectedSite} = state.map;
+
+    return {selectedSite};
+}
+
+export default connect(mapStateToProps, {})(MVUMInspectorScreen);
```

Comment:
GH-104 -- This screen is ready to go, and can cleanly display external pdfs (such as https://whatever, but I can't yet get it to work with local ones that we have on the app. We'll come back to it...